apps/charterafrica/src/payload/blocks/Spotlight.js

```diff
@@ -0,0 +1,96 @@
+import link from "../fields/link";
+
+const Spotlight = {
+  slug: "spotlight",
+  fields: [
+    {
+      name: "items",
+      type: "array",
+      minRows: 1,
+      required: true,
+      fields: [
+        {
+          name: "category",
+          type: "text",
+          label: {
+            en: "Category",
+            fr: "Catégorie",
+            pt: "Categoria",
+          },
+          required: true,
+          localized: true,
+        },
+        {
+          name: "item",
+          type: "group",
+          label: {
+            en: "Item",
+            fr: "Objet",
+            pt: "Item",
+          },
+          required: true,
+          fields: [
+            {
+              name: "title",
+              type: "text",
+              label: {
+                en: "Title",
+                fr: "Titre",
+                pt: "Título",
+              },
+              required: true,
+              localized: true,
+            },
+            {
+              name: "image",
+              label: {
+                en: "Image",
+                fr: "Image",
+                pt: "Imagem",
+              },
+              type: "relationship",
```

Comment:
```suggestion
              type: "upload",
```

And add [filtering options](https://payloadcms.com/docs/fields/upload#filtering-upload-options) to ensure we're uploading an image.

Reply:
This doesn't appear fixed @kelvinkipruto

Reply:
@kelvinkipruto [this](https://github.com/CodeForAfrica/ui/blob/829f3671cf40d877f8d19fb90c753b98ea91b625/apps/charterafrica/src/payload/blocks/Spotlight.js#L54-L56) `filterOptions`

Reply:
Yes @kelvinkipruto , you've added `filterOptions` _without_ changing the `type`. `type` should be changed to `upload`. Re-read my comment.

![Screenshot from 2023-01-16 17-02-26](https://user-images.githubusercontent.com/1779590/212696638-c5962d73-522d-4002-ae3f-1547aa56c917.png)